server/gitrest/src/routes/git/refs.ts

```diff
@@ -36,6 +40,7 @@ async function getRef(
     owner: string,
     repo: string,
     refId: string,
+    getRefParams: IGetRefParamsExternal,
```

Comment:
(nit) Line 53 is expecting this to be potentially `undefined`, should this param's type reflect that? (e.g. `getRefParams: IGetRefParamsExternal | undefined`). Alternatively, committing to it being defined would be good for consistency sake IMO